Issue with tests

<!--- Verify first that your issue/request has not already been reported -->

##### Issue type
<!--- Pick one below and delete the rest: -->
 - Bug Report

##### OverPy version
<!--- Paste verbatim output from “python -c 'import overpy; print(overpy.__version__)'” or the hash of the git commit between quotes below. -->
```
0.3.1 and pre 0.4.0
```

##### OS
- all

#### Python version
<!-- Pick at least one below and delete the rest -->

 - all

##### Summary
<!--- Explain the problem briefly. -->

From time to time the test is unable to lunch the internal web server to test the requests. This results in an error message.
```
E               urllib.error.URLError: <urlopen error [Errno 32] Broken pipe>
```

##### Steps to reproduce
<!---
Try to describe the steps how to reproduce the issue.
-->

Run a test

<!--- You can also add links to gist.github.com. -->

##### Expected results
<!--- What did you expect to happen? -->

No error

##### Actual results
<!--- What actually happened? -->

--- a/tests/__init__.py
+++ b/tests/__init__.py
@@ -1,6 +1,7 @@
 import os
 import sys
-from threading import Thread, Lock
+from multiprocessing import Process
+from threading import Lock
 
 PY2 = sys.version_info[0] == 2
 if PY2:
@@ -43,5 +44,5 @@ def new_server_thread(handle_cls, port=None):
     )
     return (
         "http://%s:%d" % (HOST, port),
-        Thread(target=server_thread, args=(server,))
+        Process(target=server_thread, args=(server,))
     )

